scripts/sdk-create-pr.sh

```diff
@@ -29,8 +29,12 @@ fi
 # Delete temp directory on exit
 trap "rm -rf ${work_dir}" EXIT
 
-# Where the git repo will be created
-mkdir ${work_dir}/git_repo
+mkdir ${work_dir}/git_repo    # Where the git repo will be created
+mkdir ${work_dir}/sdk_to_push # Copy of SDK to push
+
+# Prepare SDK to push
+cp -a ${SDK_REPO_LOCAL_PATH}/. ${work_dir}/sdk_to_push
+rm -rf ${work_dir}/sdk_to_push/.git
```

Comment:
Why cant we just copy the whole `SDK_REPO_LOCAL_PATH` to `${work_dir}/git_repo` like before? I dont see why we need to remove the `.git` file now

Reply:
Before it was being removed at the end of `make generate-sdk`.

If you don't remove the `.git` folder, when you copy the whole `SDK_REPO_LOCAL_PATH` you'll override the git definitions. That's very messy: the branch you're pushing to will be different, the git definitions are overridden, etc.

Reply:
Before it was being removed at the end of `make generate-sdk`.

If you don't remove the `.git` folder, when you copy the whole `SDK_REPO_LOCAL_PATH` you'll override the git definitions. That's very messy: the branch you're pushing to will be different, the git definitions are overridden, etc.